.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\yarin\\.virtualenvs\\SJMaster-w3yjJ9fd\\Scripts\\python.exe"
```

Comment:
I think you pushed this file by mistake?

Reply:
yes by git  add .

Reply:
Ok, are you planning to remove it?